src/interface/middleware/canUpload.js

```diff
@@ -1,12 +1,13 @@
-const getStorageUse = require('../../domain/limit/getStorageUse');
-const ErrorHandler = require('../../infra/errorHandler');
+const getStorageUse = require("../../domain/limit/getStorageUse");
+const ErrorHandler = require("../../infra/errorHandler");
 
 async function canUpload(req, res, next) {
   const invokerAddress = req.invokerAddress;
   const contractAddress = req.contractAddress;
   if (req.isAuthenticated) {
     const limit = await getStorageUse({ contractAddress });
-    if (limit.storageUse < limit.storageLimit) {
+    const totalAllowedStorage = limit.storageLimit + limit.extraStorage;
```

Comment:
We should handle the case if this value is null or undefined.

Reply:
Null values are handled in `getStorageUse` function